### DISCUSSION TITLE
jumping to numbers

### DISCUSSION THREAD

**Discussion by azinsharaf** [ggandor/leap.nvim#125] [Ideas]
Is there any option available in leap.nvim that allows you to jump to numbers? It seems it just recognizes alphabet characters. For instance if I want to jump to number `4` here:
```
- workspace: 4
   name: "4th workspace"
   status: active
```

**Comment by ggandor** [comment]
> It seems it just recognizes alphabet characters

It should jump to numbers too, it's irrelevant what the characters are. What happens if you press `s4<space>`?
